challenges-webapp/src/main/webapp/vue-app/challenges/components/AnnouncementDrawer.vue

```diff
@@ -18,7 +18,7 @@
       </div>
       <hr class="ml-4 mr-4 separator">
       <div class="pl-4 pr-4 mt-7 descriptionLabel">
-        {{ $t('challenges.label.assignedAchievement') }}
+        * {{ $t('challenges.label.assignedAchievement') }}
```

Comment:
```suggestion
        {{ $t('challenges.label.assignedAchievement') }} *
```
The mandatory indication goes after the field name.